storage/tools/log_client/main.go

```diff
@@ -6,14 +6,16 @@ import (
 	"time"
 
 	"github.com/google/trillian"
-	"github.com/google/trillian/storage/tools"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 )
 
-var treeIDFlag = flag.Int64("treeid", 3, "The tree id to use")
-var startLeafFlag = flag.Int64("start_leaf", 0, "The first leaf index to fetch")
-var numLeavesFlag = flag.Int64("num_leaves", 1, "The number of leaves to fetch")
+var (
+	treeIDFlag     = flag.Int64("treeid", 3, "The tree id to use")
+	serverPortFlag = flag.Int("port", 8090, "Port to serve log requests on")
```

Comment:
(pre-existing)  The help text is a bit misleading, as this binary isn't serving.  Maybe "Port on localhost that is serving log requests" ?